io.gitlab.liferooter.TextPieces.gnome.yaml

```diff
@@ -0,0 +1,47 @@
+# SPDX-FileCopyrightText: 2023 Gleb Smirnov <glebsmirnov0708@gmail.com>
+#
+# SPDX-License-Identifier: CC0-1.0
+
+id: io.gitlab.liferooter.TextPieces.gnome
+
+runtime: org.gnome.Platform
+runtime-version: '46'
+
+sdk: org.gnome.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.rust-stable
+
+command: textpieces-gnome
+
+finish-args:
+- --socket=fallback-x11
+- --socket=wayland
+- --device=dri
+- --env=RUST_LOG=textpieces_gnome=debug
+- --env=G_MESSAGES_DEBUG=none
+- --env=RUST_BACKTRACE=full
+- --talk-name=org.freedesktop.Flatpak
```

Comment:
this is not allowed by default. the linter will stop it.

Reply:
Do you mean `--talk-name=org.freedesktop.Flatpak`?

Reply:
what's ambiguous? 
![image](https://github.com/flathub/flathub/assets/114441/a10b688c-77ca-448b-99e6-c026c28ea940)

Reply:
> what's ambiguous? ![image](https://private-user-images.githubusercontent.com/114441/326647153-a10b688c-77ca-448b-99e6-c026c28ea940.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ0NDQ4NjksIm5iZiI6MTcxNDQ0NDU2OSwicGF0aCI6Ii8xMTQ0NDEvMzI2NjQ3MTUzLWExMGI2ODhjLTc3Y2EtNDQ4Yi05OWU2LWMwMjZjMjhlYTk0MC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQzMFQwMjM2MDlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04NmRmOTU1MmZiMTQxYmQ0NWQ3ZWFjNmE2N2Q5YmJkNWY0Y2Q2MDQ3ZGE0MDQ3OTQ4ZWRhNzZhMjJjNGQ4OGJiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.UWzRedc7SeIpQjpx8QFqijlz831mHoD3WpCf1nUEEas)

![image](https://github.com/flathub/flathub/assets/54807745/0b422ae1-ecd8-4555-bda0-8e173135284c)
This.

But ok, if you're talking about `--talk-name=org.freedesktop.Flatpak`, I wonder how am I supposed to escape the sandbox without this permission.

Reply:
it is not ambiguous. the comment is below the subject. like all review comments.

Reply:
Sorry, I'm not used to GitHub specifics since I use GitLab most of my time :)

Reply:
So what should I use instead of `--talk-name=org.freedesktop.Flatpak`?

Reply:
https://docs.flathub.org/docs/for-app-authors/linter#finish-args-flatpak-spawn-access

like it is done for the package you want to replace.